python/lsst/ts/linearstage/controllers/igus_dryve/igus_dryve.py

```diff
@@ -77,23 +76,25 @@ def __init__(self, simulation_mode, log) -> None:
         self.mode_num = None
 
         self.simulation_mode = bool(simulation_mode)
+        self.configure(config)
 
         self.log.debug(
             f"Initialized IgusLinearStageStepper, simulation mode is {self.simulation_mode}"
         )
 
     def configure(self, config):
-        self.port = config.socket_port
-        self.address = config.socket_address
+        self.config = types.SimpleNamespace(**config["stage"])
+        self.port = self.config.socket_port
+        self.address = self.config.socket_address
 
         # feed rate *must* be in mm per rotation
-        self.feed_constant = config.feed_rate
-        self.homing_speed = config.homing_speed
-        self.homing_acceleration = config.homing_acceleration
-        self.homing_timeout = config.homing_timeout
-        self.motion_speed = config.motion_speed
-        self.motion_acceleration = config.motion_acceleration
-        self.maximum_stroke = config.maximum_stroke
+        self.feed_constant = self.config.feed_rate
+        self.homing_speed = self.config.homing_speed
+        self.homing_acceleration = self.config.homing_acceleration
+        self.homing_timeout = self.config.homing_timeout
+        self.motion_speed = self.config.motion_speed
+        self.motion_acceleration = self.config.motion_acceleration
+        self.maximum_stroke = self.config.maximum_stroke
```

Comment:
I *strongly* suggest not copying the config. Just save the config and get these properties from self.config when you need them. Less duplication, just one way to do things, and robust against config tweaks.